users/luescher/setups/hybrid/util.py

```diff
@@ -0,0 +1,421 @@
+            'warp_freqbreak': None,  # 3700
+            'tempint_type': 'hanning',
+            'tempint_shift': .01,
+            'tempint_length': .025,
+            'flush_before_gap': True,
+            'do_specint': False,
+            'specint_type': 'hanning',
+            'specint_shift': 4,
+            'specint_length': 9,
+            'normalize': True,
+            'preemphasis': True,
+            'legacy_scaling': False,
+            'without_samples': False,
+            'samples_options': {'audio_format': "wav",
+                                'dc_detection': True},
+            'normalization_options': {},
+          }
+          'fb':
+          {
+            'warping_function': "mel",
+            'filter_width': 80,
+            'normalize': True,
+            'normalization_options': None,
+            'without_samples': False,
+            'samples_options': {'audio_format': "wav",
+                                'dc_detection': True},
+            'fft_options': None,
+            'apply_log': True,
+            'add_epsilon': False,
+          }
+          'energy':
+          {
+              'without_samples': False,
+              'samples_options': {'audio_format': "wav",
+                          'dc_detection': True},
+              'fft_options': {},
+          }
+        ##################################################
+        :param linear_alignment_args:
+          'minimum_segment_length': 0,
+          'maximum_segment_length': 6000,
+          'iterations': 20,
+          'penalty': 0,
+          'minimum_speech_proportion': .7,
+          'save_alignment': False,
+          'keep_accumulators': False,
+          'extra_merge_args': None,
+          'extra_config': None,
+          'extra_post_config': None,
+        ##################################################
+        :param monophone_training_args:
+          'feature_flow': 'mfcc+deriv+norm',
+          'feature_energy_flow': 'energy,mfcc+deriv+norm',
+          'align_iter': 75,
+          'splits': 10,
+          'accs_per_split': 2,
+        :param monophone_recognition_args:
+          'eval_iter': [7, 8, 9, 10]
+          'pronunciation_scales': [10.0, 11.0, 12.0, 13.0, 14.0]
+          'lm_scales': [9.0, 9.25, 9.50, 9.75, 10.0, 10.25, 10.50]
+          'recog_args': {
+                          'feature_flow': dev_corpus_name,
+                          'pronunciation_scale': pronunciation_scale,
+                          'lm_scale': lm_scale,
+                          'lm_lookahead': True,
+                          'lookahead_options': None,
+                          'create_lattice': True,
+                          'eval_single_best': True,
+                          'eval_best_in_lattice': True,
+                          'search_parameters': {
+                            'beam_pruning': 18.0,
+                            'beam-pruning-limit': 100000,
+                            'word-end-pruning': 0.75,
```

Comment:
Please do NOT suggest 0.75 here. @curufinwe can provide a better value. (My guess would be 0.5)